packages/data-store/src/types/uniformCredential/IAbstractUniformCredentialStore.ts

```diff
@@ -0,0 +1,29 @@
+import { CredentialCorrelationType, CredentialDocumentFormat, CredentialStateType, CredentialTypeEnum, UniformCredential } from './uniformCredential'
+
+export type GetUniformCredentialArgs = { id: string } | { hash: string }
+
+export type FindUniformCredentialArgs = Array<Partial<UniformCredential>>
+
+// TODO: discuss about what args we want here
+export type GetUniformCredentialsArgs = {
+  filter?: FindUniformCredentialArgs
+}
+
+export type AddUniformCredentialArgs = {
+  credentialType: CredentialTypeEnum
+  documentFormat: CredentialDocumentFormat
+  raw: string
+  issuerCorrelationType: CredentialCorrelationType
+  subjectCorrelationType?: CredentialCorrelationType
+  issuerCorrelationId: string
+  subjectCorrelationId?: string
+  tenantId?: string
+  expiresAt?: Date
+  state?: CredentialStateType
```

Comment:
I mentioned verificationState for a reason, because the other related property is called verificationDate (which btw is a poor name, because for verification the actual time is also pretty important, contrary to issuance and revocation. For the latter there of course also is a time associated but these are typically important at a day granularity. Having said that, for VCDM 2.0 the names have moved to validFrom and validUntil, so it is probably best to already use these for our DB columns as well. https://www.w3.org/TR/vc-data-model-2.0/#validity-period

So either call both verification of verified, but do not mix them. So suggest to call it `verifiedState` and `verifiedAt` or something. One is about when it happened, the other is the outcome at that specific point in time. We mainly need these for fast and easy access in the UI

Reply:
changed the `lastVerificationDate` to `verifiedAt` also did that for `revocationDate` and changed it to `revokedAt`